app/src/main/res/values/strings.xml

```diff
@@ -485,6 +485,10 @@
          and is used to customize if we use passthrough mode when the app starts. -->
     <string name="display_options_start_with_passthrough">Start with Passthrough Mode</string>
 
+    <!-- This string labels an On/Off switch in the 'Display Options' dialog and is used to
+         customize if windows can be moved. Enabling it adds a Move button to the navigation bar. -->
+    <string name="display_options_windows_can_move">Windows can be moved</string>
```

Comment:
I'd say something "Drag to move windows"

Reply:
Updated